src/openforms/config/tests/test_csp_update.py

```diff
@@ -27,3 +41,87 @@ def test_middleware_applies_cspsetting_models(self):
         self.assertIn("http://foo.bar", csp_policy["img-src"])
         self.assertIn("http://bazz.bar", csp_policy["img-src"])
         self.assertIn("http://buzz.bazz", csp_policy["default-src"])
+
+
+class CreateCSPFormActionFromConfigTests(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.addCleanup(clear_caches)
+
+    def test_no_ogone_nor_digid_eherkenning_config(self):
+        assert not OgoneMerchant.objects.exists()
```

Comment:
Why don't we use `self.assertFalse` instead of `assert not`. This is a general question because I see it quite often in the tests.

Reply:
so the `self.assertFOO` is to really do the test assertion - it makes it clear that this is the part that you are testing

the `assert` at the beginning of the test is more of a sanity check - did I set up my test data correctly? it makes certain implicit behaviours explicit. Here the assumption is that no records exists at all at the beginning of the test, such that calling the management command on empty data does not produce anything.

I actually had these assertions fail locally because of broken test isolation with other tests, and it helps in tracking down test failures that you can otherwise not explain easily.

So to summarize:

* assert at the beginning: sanity check for test data, not part of the test itself
* self.assertFOO: the actual meaningful part of the test